index.js

```diff
@@ -1,18 +1,50 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
-console.log("Here's your header:", header)
+let header = document.querySelector("h1#header")
+console.log("Here's your header:", header.textContent)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = "red"
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
 
+let travelerSpecs = document.querySelector("#profile")
+h2ToUpdate = travelerSpecs.querySelector("h2")
+h2ToUpdate.textContent = traveler.name
 
-/***** Deliverable 4 *****/
+emToUpdate = travelerSpecs.querySelector("em")
+emToUpdate.textContent = traveler.nickname
+
+imgToUpdate = travelerSpecs.querySelector("img")
+imgToUpdate.src = traveler.photo
+imgToUpdate.alt = traveler.name
 
 
+/***** Deliverable 4 *****/
+
+traveler.animalSightings.forEach( function(animalSighting) {
+    let li = document.createElement('li')
+    let p = document.createElement('p')
+    let img = document.createElement('img')
+    let a = document.createElement('a')
+    //
+    li.id = animalSighting.id 
```

Comment:
For this element, the goal is to set the `data-id` attribute, not the `id` attribute. You can set the `data-id` attribute like this:

```js
    li.dataset.id = animalSighting.id 
```

That will create an element like this:

```html
<li data-id="1">
```